osctiny/extensions/packages.py

```diff
@@ -64,13 +65,22 @@ def get_list(self, project: str, deleted: bool = False, expand: bool = False, **
         .. versionchanged:: 0.7.6
             Changed default value of ``expand`` to ``False``
 
+        .. versionchanged:: {{NEXT_RELEASE}}
+            Made the ``deleted`` parameter fully optional. It is no longer a boolean flag but really
+            a boolean parameter, so ``deleted=0`` is actually added to the API call.
+
         :param project: name of project
-        :param deleted: Show deleted packages instead
+        :param deleted: If true, shows deleted packages instead of the current ones. If False is
+                        passed, this causes a bypass in the rails backend of the build service,
+                        returning results from the file backend directly. Default is None.
```

Comment:
Technically not true as the query is passed to the backend directly in all cases:
```suggestion
        :param deleted: If true, also shows deleted packages instead of the present ones.
                        If False, then `deleted=0` is appended to the query explicitly.
                        Default is `None`, which implies no parameter addition to the query.
```